apps/src/javalab/JavalabEditor.jsx

```diff
@@ -389,11 +401,13 @@ class JavalabEditor extends React.Component {
     newTabs[index1] = newTabs[index2];
     newTabs[index2] = file1;
     setOrderedTabKeys(newTabs);
+    this.updateFileOrder();
     // closes the tab menu if it is open
```

Comment:
This is being handled by `updateFileOrder` now right?

Reply:
updateFileOrder calls on sourceFileOrderUpdated which updates the tabOrder of files in sources according to orderedTabKeys that was just modified by the swapping of index1 and index2.

Reply:
Oh I meant closing the tab menu - looks like updateFileOrder does that internally right?

Reply:
Oh yes! It does - thanks! Will remove.

Reply:
Update: Removed the closing of the tab menu from updateFileOrder to narrow scope of what updateFileOrder does.